src/small_vec.rs

```diff
@@ -0,0 +1,201 @@
+const WORD_SIZE: usize = 64;
```

Comment:
I think you can derive this from `usize::max_value()` (probably something like `log(usize::max_value()) + 1` but I might be wrong).

Reply:
That is a good point. However, the compiler is complaining that `const fns are an unstable feature` and I don't know a way around this error.

Reply:
Ah, you'll probably need to use a macro I guess. [I think the "const fns" might be a red herring: we don't want to make the function constant, but it results.] A simple macro for this is probably doable, but if it looks too complicated, feel free to keep things as they are.

Reply:
I'll look into it